hugo/data/home/roadmap.yml

```diff
@@ -1,3 +0,0 @@
-roadmap:
```

Comment:
So partials can use the whole home yml and just pass that instead of the global context. In particular, the roadmap partial uses as well some things that are in the general data file.
If the data in roadmap was big enough, like with projects and testimonials to be on its own, I'd have left it.